src/sentry/models/pullrequest.py

```diff
@@ -87,3 +88,19 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_pullrequest_commit"
         unique_together = (("pull_request", "commit"),)
+
+
+@region_silo_only_model
```

Comment:
For my own learning, how do you decide between region and control?

Reply:
data is generally stored on region silos is how i think about it. user and auth tables are in the control silo

Reply:
Most tables are in regions. Almost all data that is related to a single organization is a region model. Control silo tables are things shared across or between organizations. Some examples of that are users (emails, notification settings, api tokens), organization slugs (so we can find what region an organization is in), integration configuration, and system records like audit logs (we'd like them centrally queryable), broadcasts, policy documents. If you're ever need help deciding drop by #proj-hybrid-cloud and we'll help out.